### PR TITLE
integration tests - swap EXPECT and INPUT (optional INPUT)

### PR DESCRIPTION
**Pull request by gharlan**

### CODE REVIEW

File: Symfony/CS/Tests/AbstractIntegrationTest.php [hunk 1]
```diff
@@ -36,16 +36,16 @@
...
+ * Expected code after fixing
+ * --INPUT--
+ * Code to fix*****
  *
  *     * Additional fixers may be omitted.
  *    ** PHP minimum version. Default to current running php version (no effect).
  *   *** HHVM compliant flag. Default to true. Set to false to skip test under HHVM.
  *  **** Black listed filters may be omitted.
- * ***** When the expected block is omitted the input is expected not to
+ * ***** When the input block is omitted the expected part is expected not to
```
Comment (keradus) [inline Symfony/CS/Tests/AbstractIntegrationTest.php]: it is true even when input block is present

Comment (gharlan) [inline Symfony/CS/Tests/AbstractIntegrationTest.php]: good point.

"When the input block is omitted only the expected parts is checked to not be changed by the fixers."

Ok? Other proposal?

Comment (keradus) [inline Symfony/CS/Tests/AbstractIntegrationTest.php]: I would just say it is optional

**Comment by keradus** [comment]
:+1:

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by SpacePossum** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @gharlan.

**Pull request merged**